emperor/support_files/js/view.js

```diff
@@ -41,6 +41,13 @@ function DecompositionView(decomp) {
    */
   this.visibleDimensions = [0, 1, 2]; // We default to the first three PCs
   /**
+   * Orientation of the axes, `-1` means the axis is flipped, `1` means the
+   * axis is not flipped.
+   * @type {integer[]}
+   * @default [1, 1, 1]
+   */
+  this.axesOrientation = [1, 1, 1];
```

Comment:
Thanks. Just for completeness, I also thought about that, but it was
somewhat more innatural.

On (Jul-27-16|13:56), Jose Navas wrote:
> > @@ -41,6 +41,13 @@ function DecompositionView(decomp) {
> >     */
> >    this.visibleDimensions = [0, 1, 2]; // We default to the first three PCs
> >    /**
> > +   * Orientation of the axes, `-1` means the axis is flipped, `1` means the
> > +   * axis is not flipped.
> > +   * @type {integer[]}
> > +   * @default [1, 1, 1]
> > +   */
> > +  this.axesOrientation = [1, 1, 1];
> 
> Ooohh nevermind - I just saw when they're used!
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/525/files/006b089d818c8da741fc622182e71915bf102fe7#r72521910